c/configmgr.c

```diff
@@ -682,11 +709,28 @@ static Json *readYamlIntoJson(ConfigManager *mgr, char *filename, bool allowMiss
   } else if (allowMissingFile && wasMissing){
     return NULL;
   } else{
-    trace(mgr,INFO,"WARNING, yaml read failed, errorBuffer='%s'\n",errorBuffer);
-#ifdef __ZOWE_OS_ZOS
-    a2e(errorBuffer,YAML_ERROR_MAX);
+#ifndef __ZOWE_OS_ZOS
+    /*
+     * Let's convert to ascii if we aren't on z/OS?
+     */
+    e2a(errorBuffer,YAML_ERROR_MAX);
```

Comment:
Use `sizeof(errorBuffer)` since in case someone changes `char errorBuffer[YAML_ERROR_MAX];` to, for example, `char errorBuffer[YAML_ERROR_MAX + 1];`, this would cause issues.

Reply:
(Fixed).